front/src/modules/scenario/components/ScenarioCard.tsx

```diff
@@ -6,12 +6,13 @@ import infraLogo from 'assets/pictures/components/tracks.svg';
 import { AiFillFolderOpen } from 'react-icons/ai';
 import { dateTimeFormatting } from 'utils/date';
 import { useDispatch } from 'react-redux';
-import { updateScenarioID } from 'reducers/osrdconf';
 import { updateSelectedProjection } from 'reducers/osrdsimulation/actions';
 import { FcCalendar } from 'react-icons/fc';
 import { MdTrain } from 'react-icons/md';
 import nextId from 'react-id-generator';
 import { ScenarioWithCountTrains } from 'common/api/osrdEditoastApi';
+import { useOsrdActions } from 'common/osrdContext';
+import { operationalStudiesConfSliceActionsType } from 'reducers/osrdconf/operationalStudiesConf';
```

Comment:
Could you add `type` to all other type imports of the file please :)
```suggestion
import type { operationalStudiesConfSliceActionsType } from 'reducers/osrdconf/operationalStudiesConf';
```

Reply:
Done